internal/model/model.go

```diff
@@ -1102,13 +1102,19 @@ func (m *Model) ScanFolders() map[string]error {
 }
 
 func (m *Model) ScanFolder(folder string) error {
-	return m.ScanFolderSub(folder, "")
+	return m.ScanFolderSubs(folder, nil)
 }
 
-func (m *Model) ScanFolderSub(folder, sub string) error {
-	sub = osutil.NativeFilename(sub)
-	if p := filepath.Clean(filepath.Join(folder, sub)); !strings.HasPrefix(p, folder) {
-		return errors.New("invalid subpath")
+func (m *Model) ScanFolderSubs(folder string, subs []string) error {
+	if subs == nil {
+		subs = []string{""}
+	}
```

Comment:
Could skip this and just do add the `if len(subs) == 0 ...` down around the `hasPrefix` stuff, which I think is the only place it's required?